src/components/PushNotificationLayout.js

```diff
@@ -0,0 +1,68 @@
+import * as firebase from "firebase/app";
+import { useRouter } from "next/router";
+import React, { useEffect } from "react";
+import { toast, ToastContainer } from "react-toastify";
+import "firebase/messaging";
+
+import { firebaseCloudMessaging } from "../utils/firebase";
+
+function PushNotificationLayout({ children }) {
+    const router = useRouter();
+    useEffect(() => {
+        setToken();
+
+        // Event listener that listens for the push notification event in the background
+        if ("serviceWorker" in navigator) {
+            navigator.serviceWorker.addEventListener("message", (event) => {
+                console.log("event for the service worker", event);
+            });
+        }
+
+        // Calls the getMessage() function if the token is there
+        async function setToken() {
+            try {
+                const token = await firebaseCloudMessaging.init();
+                if (token) {
+                    console.log("token", token);
```

Comment:
- Remove the log statement 💡

Reply:
This file is unused, so i remove it from project